pkgs/tools/misc/zsh-history-to-fish/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonApplication, fetchPypi, click }:
+
+buildPythonApplication rec {
+  pname = "zsh-history-to-fish";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-expPuffZttyXNRreplPC5Ee/jfWAyOnmjTIMXONtrnw=";
+  };
+
+  nativeBuildInputs = [ ];
+  propagatedBuildInputs = [ click ];
```

```suggestion
  propagatedBuildInputs = with python3.pkgs; [
    click
  ];
```
We can use `python3.pkgs` again.